src/_guides/language/effective-dart/documentation.md

```diff
@@ -372,21 +372,6 @@ a flavor of what's supported:
 ///     * They must be indented at least 4 spaces.
 ///     * (Well, 5 including the space after `///`.)
 ///
-/// Code blocks are fenced in triple backticks:
-///
-/// ```
-/// this.code
-///     .will
-///     .retain(its, formatting);
-/// ```
-///
-/// The code language (for syntax highlighting) defaults to Dart. You can
-/// specify it by putting the name of the language after the opening backticks:
-///
-/// ```html
-/// <h1>HTML is magical!</h1>
-/// ```
-///
```

Comment:
It would be good not to lose these examples. Now that @chalin fixed the excerpt generator, can you put these back in?

Reply:
He added them back in his PR. It turned out to be an issue with the closing tag.

Reply:
👍